_pkgdown.yml

```diff
@@ -68,6 +68,13 @@ reference:
     - iic
     - poisson_log_loss
 
+  - title: Fairness Metrics
+    contents:
+    - fairness_metric
```

Comment:
this function should be listed under `Development Functions`

Reply:
It's significantly higher level than the functions for creating new metrics, and I think we suspect more likely to be used by "regular" users.